checkbox-ng/checkbox_ng/launcher/master.py

```diff
@@ -503,6 +505,14 @@ def finish_session(self):
                 tmp_sig = signal.signal(signal.SIGINT, signal.SIG_IGN)
                 stack.callback(signal.signal, signal.SIGINT, tmp_sig)
                 self._export_results()
+        # let's see if any of the jobs failed, if so, let's return an error code of 1
+        job_state_map = (
+            self._sa.manager.default_device_context._state._job_state_map
+        )
+        self._has_anything_failed = any(
+            job.result.outcome == "fail" for job in job_state_map.values()
```

Comment:
what about crashed jobs? (i know that somewhere in checkbox there is a mapping that is done to map every possible state to one of `pass/skip/fail`, I'm just not sure if it's the case here)

Reply:
OK I checked by running a test plan that only has one job that's crashing, and checkbox retrurns 0. Oops :)

Reply:
Yeah, very valid concern.
I think it can be done as a followup as the code provided here is critical to land fast.